sys/shell/commands/sc_icmpv6_echo.c

```diff
@@ -168,6 +197,9 @@ int _icmpv6_ping(int argc, char **argv)
     if (argc > (3 + param_offset)) {
         _a_to_timex(&delay, argv[3 + param_offset]);
     }
+    if (argc > (4 + param_offset)) {
```

Comment:
can you add this additional parameter also to the usage text that is printed when `ping6` is invoked without any arguments?